doc/articles/features/routed-events.md

```diff
@@ -159,19 +159,17 @@ These _routed events_ are not implemented yet in Uno:
 * `DragOver`
 * `Drop`
 * `Holding`
-* `ManipulationCompleted`
-* `ManipulationDelta`
 * `ManipulationInertiaStarting`
-* `ManipulationStarted`
-* `ManipulationStarting`
```

Comment:
You remvoed them from the "unsupported" section, but they should appear now in the "supported" section :-)

Reply:
Indeed, I marked them as implemented at the bottom of the file when I did it, but I forgot to update the table at the top of the file ... done!